tests/features/capsulahub-application.feature

```diff
@@ -0,0 +1,182 @@
+    And   one of the following values for <token> is provided
+          |<token>     |
+          |''        |
+          |{}        |
+          |{ test: 'test' }|
+          |[]        |
+          |['test']  |
+          |null      |
+          |undefined |
+          |true      |
+          |false     |
+          |0         |
+          |-1        |
+    Then  a relevant error is received
+
+  Scenario: Run Capsula-Hub instance with invalid port
+    Given a valid token with configuration
+    When  I run the command `capsula-hub run --token="token" --port="port" `
+    And   one of the following values for <port> is provided
+          |<port>    |
+          | 'abc'    |
+          | 65536    |
+          | 0        |
+          |{}        |
+          |{ test: 'test' }|
+          |[]        |
+          |['test']  |
+          |null      |
+          |undefined |
+          |true      |
+          |false     |
+          |-1        |
+    Then  a relevant error is received
+
+  Scenario: Run Capsula-Hub instance with an invalid value of <configProvider>
+    Given a valid token with configuration
+    When  I run the command `capsula-hub run --token="token" --configProvider="configProvider" `
+    And   one of the following values for <configProvider> is provided
+          |<configProvider>    |
+          | 'abc'    |
+          | 65536    |
+          | 0        |
+          |{}        |
+          |{ test: 'test' }|
+          |[]        |
+          |['test']  |
+          |null      |
+          |undefined |
+          |true      |
+          |false     |
+          |-1        |
+    Then  a relevant error is received
+
+  Scenario: Run Capsula-Hub instance with an non-existent configProvider throws an error
+    Given a valid token with configuration
+    When  I run the command `capsula-hub run --token="token" --configProvider="configProvider" `
+    And   "configProvider" is not in the list of available configuration types
+    Then  a relevant error is received
+
+  Scenario: Run Capsula-Hub instance twice on the same port
+    Given a valid token with configuration
+    And   a valid port
+    When  I run the command `capsula-hub run --token="token" --port="port" `
+    And   I run the same command again
+    Then  a relevant error is received
+
+  Scenario: Run Capsula-Hub  instance with the same token for different configProvider on two different ports
+    Given a valid token with configuration
+    And   two valid ports "port1" and "port2"
+    When  I run the command `capsula-hub run --token="token" --configProvider="hardcoreServer" --dispatcherUrl="dispatcherUrl" --port="port1 "`
+    And   I run the command `capsula-hub run --token="token" --port="port2"`
+    Then  an workspace instance is created that is running on port "port1"
+    And   hardcoreServer is the configuration provider used
+    And   an workspace instance is created that is running on port "port2"
+    And   httpFile is the configuration provider used
```

Comment:
It makes sense to have a test, where capsulahub runs a few times with different tokens on different ports - to make sure that a different configurations are applied